### PR TITLE
translation for createContext

### PR DESCRIPTION
**Pull request by r17x**
Translation for `createContext` https://github.com/reactjs/id.react.dev/issues/330

### CODE REVIEW

**Comment by r17x** [comment]
Ready to review @resir014

File: src/content/reference/react/createContext.md [hunk 1]
```diff
@@ -196,22 +196,22 @@ function App() {
...
 
-Code like this specifies the *default* context value:
+Kode seperti ini menentukan nilai konteks *default*:
 
 ```js
 const ThemeContext = createContext('light');
 ```
 
-This value never changes. React only uses this value as a fallback if it can't find a matching provider above.
+Nilai ini tidak pernah berubah. React hanya menggunakan nilai ini sebagai fallback jika tidak dapat menemukan penyedia yang cocok di atas.
```
Comment (mazipan) [inline src/content/reference/react/createContext.md:214]: Kalau mau pakai `fallback`, kasih miring aja. `*fallback*`.

File: src/content/reference/react/createContext.md [hunk 2]
```diff
@@ -150,15 +150,15 @@ function App() {
 }
 ```
 
-Now the `Page` component and any components inside it, no matter how deep, will "see" the passed context values. If the passed context values change, React will re-render the components reading the context as well.
+Sekarang komponen `Page` dan semua komponen didalamnya, tidak peduli seberapa dalam, akan "melihat" nilai konteks yang diberikan. Jika nilai konteks yang diberikan berubah, React akan me-_render_ulang komponen yang membaca konteks tersebut.
```
Comment (mazipan) [inline src/content/reference/react/createContext.md:153]: Ini gua masih gak tau kita mau ambil yang mana.
Sama kaya komenku di PR nya @resir014.
Kita mau pakai `me-render` apa `merender` atau bahkan `me*render*`.
Btw ini cuma kata `render` aja yang rancu, kata `ulang` nya kan gak ada masalah.

File: src/content/reference/react/createContext.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ const ThemeContext = createContext('light');
 const AuthContext = createContext(null);
 ```
 
-`createContext` returns a <CodeStep step={1}>context object</CodeStep>. Components can read context by passing it to [`useContext()`](/reference/react/useContext):
+`createContext` mengembalikan <CodeStep step={1}>konteks objek</CodeStep>. Komponen datap membaca konteks dengan mengoper ke [`useContext()`](/reference/react/useContext):
```
Comment (mazipan) [inline src/content/reference/react/createContext.md:118]:
```suggestion
`createContext` mengembalikan <CodeStep step={1}>konteks objek</CodeStep>. Komponen membaca konteks dengan mengoper ke [`useContext()`](/reference/react/useContext):
```

File: src/content/reference/react/createContext.md (hunk 2 above)
Comment (r17x) [reply src/content/reference/react/createContext.md:153]: Sama, membuat bingung dan memang perlu ditentukan digloarium kalau udah diputusin yang mana yang mau diambil cc @resir014 @mazipan

Comment (r17x) [reply src/content/reference/react/createContext.md:153]: Oh iya, aku ambil dari glosari untuk 'render' @mazipan  lalu tambahkan me- 
https://github.com/reactjs/id.react.dev/wiki/Glosarium#r

**Comment by mhaidarhanif** [comment]
Need to confirm again whether to use `context` or `konteks` in the translation, might need to be decided and added to the Glossary.

cc @resir014 @r17x @mazipan @zainfathoni

**Comment by mhaidarhanif** [comment]
@r17x I've added some typo review comments

File: src/content/reference/react/createContext.md (hunk 2 above)
Comment (mazipan) [reply src/content/reference/react/createContext.md]: Di konteks re-render, jadi `merender ulang`.
Sekalian check semua yang kata-kata render di bawahnya ya.

cc: @resir014

File: src/content/reference/react/createContext.md [hunk 4]
```diff
@@ -168,7 +168,7 @@ export const ThemeContext = createContext('light');
 export const AuthContext = createContext(null);
 ````
 
-Components declared in other files can then use the [`import`](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import) statement to read or provide this context:
+Komponen yang dideklarasikan di berkas lain dapat menggunakan pernyataan [`import`](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import) untuk membaca atau menyediakan konteks ini:
```
Comment (fzn0x) [inline src/content/reference/react/createContext.md:171]: Perhaps the use of the word `berkas` here needs to be reconfirmed, as some remain in `file` or `files`.

**Review by fzn0x — COMMENTED** [review]
I would like to comment on the following

File: src/content/reference/react/createContext.md (hunk 2 above)
Comment (r17x) [reply src/content/reference/react/createContext.md]: Done @mazipan  aku ubah jadi merender.

File: src/content/reference/react/createContext.md (hunk 4 above)
Comment (r17x) [reply src/content/reference/react/createContext.md]: Back to file ->`*file*`

**Review by resir014 — APPROVED** [review]

**Review by mazipan — APPROVED** [review]

**Comment by mhaidarhanif** [comment]
@r17x @resir014 Seems there are still some typos and inconsistency though. We'll fix them on later PR then.

`di dalam` instead of `didalam`
`di luar` instead of `diluar`
Using `*` instead of `_` for italics

**Pull request merged**